app/controllers/games_controller.rb

```diff
@@ -1,6 +1,14 @@
 class GamesController < ApplicationController
 
   def index
+    @games = Game.all
```

Comment:
Oops just noticed your not using available scope here to filter games here

Reply:
https://github.com/scottAF/ChestMasters/pull/15/commits/0d5d841fd174ba17e1ba210d0a2d5cad89a293d3